/ndt:
  ros__parameters:
    # NDT matcher
    epsilon: 0.01
    stepsize: 0.1
    resolution: 1.0
    maxiters: 20
    use_initial_guess: true
    leafsize: 1.0

    # TF2 buffer size
    tf2_buffer_size: 10

    # Subscriber topics
    scan_topic: "/scan"
    initial_pose_topic: "/initial_pose"
    map_topic: "/map"

    # Publisher topics
    pose_topic: "/pose"
    cloud_topic: "/cloud"
    tf_topic: "/tf"

    # Reference frames
    map_frame: "map"
    vehicle_frame: "base_link"
    laser_frame: "os_sensor"

/map_publisher:
  ros__parameters:
    map_file: "/ros2_ws/install/ndt_locator/share/ndt_locator/maps/court_yard_map.pcd"
